src/docs/sphinx/developerGuide/Contributing/InstallWin.rst

```diff
@@ -0,0 +1,266 @@
+
+
+    .. code:: shell
+
+        PS> $env:VERSION='164-677'
+        PS> $env:IMG='ubuntu18.04-gcc8'
+        PS> $env:REMOTE_DEV_IMG="remote-dev-${env:IMG}"
+
+
+Please note the preposition of *env:* in the windows formalisme. The **${ORG}** variable will be hard-coded as *geosx*. The last variable will be used as
+an informative image name. Note also that the version above is the might not be the most recent version of the image at the time you are reading these lines and
+should be change for the closest to the *TPL* commit associated with the GEOSX commit you cant to work from.
+
+- You'll need to generate a ssh-key to be able to access the container without the need for defining a password. This can be done from the *PowerShell*,
+
+
+    .. code:: shell
+
+        PS > ssh-keygen.exe
+        PS > cat [path-to-gen-key]/[your-key].pub
+
+
+The first command will prompt message, asking for you to complete the deisred path for the key as well as a passphrase, with confirmation.
+More details on `ssh-key generation <https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement#user-key-generation>`_.
+
+
+4. Build the image and run the container
+-----------------------------------------
+
+The preliminary tasks are now done, let us build the image that will be containerized.
+
+.. code:: shell
+
+    PS> cd [path-to-dockerfile-folder]/
+    PS > docker build --build-arg ORG=geosx --build-arg IMG=${env:IMG} --build-arg VERSION=${env:VERSION} -t ${env:REMOTE_DEV_IMG}:${env:VERSION} -f Dockerfile
+
+As described above, we are passing our environment variables in the building stage, which offer the flwxibility of changin the version or image by a simple redefinition.
+A log updating or pulling the different layer should be displayed afterwards and on the last line the *image id*. We can check that the image is created using *PowerShell* CLI:
+
+.. code:: shell
+
+    PS > docker images
+
+or using the *Docker Desktop*
+
+.. image:: win_install/win_docker_images.png
+   :width: 45%
+
+Now that we have the image build, let us run a container from,
+
+.. code:: shell
+
+    PS > docker run --cap-add=ALL  -d --name ${env:REMOTE_DEV_IMG}-${env:VERSION} -p 64000:22 --mount 'type=bind,source=D:/install_geosx_docker/,target=/app' ${env:REMOTE_DEV_IMG}:${env:VERSION}
+
+Note that in addition to the detached flag (*-d*) and the name tage (*--name*), we provide *Docker* with the port the conatiner should associated to
+communicate with ssh port 22, as well as a binding between a host mount point (*D:/install_geosx_docker/*) and a container mount point (*/app*) to have a peristent storage
+for our development/geosx builds. More details on the `--mount options <https://docs.docker.com/storage/bind-mounts/>`_
+
+There exact same steps can be achieve using th *Docker Desktop* GUI in the image tabs, clicking on the run button and filling the same information in the interface,
```

Comment:
```suggestion
There exact same steps can be achieved using the *Docker Desktop* GUI in the image tabs, clicking on the *run* button and filling the same information in the interface,
```

Reply:
Isn't it imprecise since `--cap-add=ALL` does not seem to be accessible?

Reply:
Change for a similar. TBH Idk if `--cap--add=ALL` make a difference of use in that specific case. Both are running and available for connection.

Reply:
This is the reason why we use this `docker` option https://github.com/GEOSX/GEOSX/pull/1247

Reply:
Ok, then I have to find a way of doing that also in the GUI :-p

Reply:
Or just forget about the GUI!